### ISSUE TITLE
After the game is uninstalled and reinstalled, the playgameservices data is gone

### ISSUE THREAD

**Issue by longshaowu** [playgameservices/play-games-plugin-for-unity#3160]
PlayGamesPlatform.Instance.SavedGame.OpenWithAutomaticConflictResolution(
                    name,
		    DataSource.ReadNetworkOnly,
		    ConflictResolutionStrategy.UseLastKnownGood, onConnectionOpen);

After the game is uninstalled and reinstalled, the playgameservices data is gone. 
Only one phone appears, other phone data is still there.

**Comment by kyubuns** [comment]
I have the same problem.
I can share save data on multiple devices, but when I uninstall, the data is lost.
Is this the correct behavior?
I am using v11.01.

**Comment by Ghigolus** [comment]
Hello, I found a workaround for this issue. Try to tap on the refresh button when you reinstall the app.

**Comment by kyubuns** [comment]
> Hello, I found a workaround for this issue. Try to tap on the refresh button when you reinstall the app.

Sorry, which refresh button are you talking about?

**Comment by Ghigolus** [comment]
when you try to select saves, it opens a panel with all your saves. I think that here you cannot find any of your saved data. So, in this panel there is a refresh button in the top right corner. If you tap on it, you will be able to see your saves. You have to do this only the first time after the reinstall.

**Comment by kyubuns** [comment]
Oh, I see!
I didn't call ShowSelectSavedGameUI because I only need one saved data for my game.

I do not think this problem has been solved.
I wish we could call “refresh” from scripts...
Does anyone know of any other information?

**Comment by Ghigolus** [comment]
Yes, I would like to call some kind of Refresh() from script. This will fix my issue too. Right now I have to inform the user that he has to click on the refresh button but it is not the best way to proceed.. In your case you need to find how to refresh it automatically or wait for an update of this plugin

**Comment by KirillKuzyk** [comment]
Same issue for me.
https://github.com/playgameservices/play-games-plugin-for-unity/issues/3167#issuecomment-1159644069

**Comment by KirillKuzyk** [comment]
I think I've figured out the cause of the issue:
https://github.com/playgameservices/play-games-plugin-for-unity/issues/3169

@longshaowu @Ghigolus @kyubuns Please confirm your issue is connected to my bug report by launching the adb tool with these filters:
`adb logcat -s Unity Volley SnapshotUploaderImpl`
And check if you also have these errors:
```
E Volley  : [348] BasicNetwork.performRequest: Unexpected response code 403 for https://www.googleapis.com/games/v1/snapshotsExtended/gameSave:prepareRevision
W SnapshotUploaderImpl: {"code":403,"errors":[{"reason":"UrlLeaseLimitExceeded","domain":"usageLimits","message":"Too many pending uploads for this snapshot. Please finish or cancel some before creating more."}]}
```

**Comment by kyubuns** [comment]
I'm sorry about this.
I thought I was using DataSource.ReadNetworkOnly, but I just called it with DataSource.ReadCacheOrNetwork
I changed the code to the normal "use DataSource.ReadNetworkOnly only the first time after startup" code and the problem is solved.

**Comment by fadizant** [comment]
> I'm sorry about this. I thought I was using DataSource.ReadNetworkOnly, but I just called it with DataSource.ReadCacheOrNetwork I changed the code to the normal "use DataSource.ReadNetworkOnly only the first time after startup" code and the problem is solved.

Did the same but did not fix the problem, unfortunately.

**Comment by Kev-The-GameDev** [comment]
> > I'm sorry about this. I thought I was using DataSource.ReadNetworkOnly, but I just called it with DataSource.ReadCacheOrNetwork I changed the code to the normal "use DataSource.ReadNetworkOnly only the first time after startup" code and the problem is solved.
> 
> Did the same but did not fix the problem, unfortunately.

Hi! Have you found a solution for this? I'm trying to get this to work for days now..
